packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -23,6 +23,12 @@
 
 namespace DataTransferKit
 {
+namespace Details
+{
```

Comment:
So this is part of the namespace details but not in the details subdirectory :-/